LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 SailReal / DonatJR
```

Comment:
Should we not put our real names her inside? 
Maybe we will change our github account names sometimes in the  future?

Reply:
I created this using the github license template which uses the username automatically, but we could of course use our real names - do you want me to change it?